src/tools/ts-generator/main.cpp

```diff
@@ -127,7 +125,7 @@ int main(int argc, char *argv[])
     }
 
     study->initializeRuntimeInfos();
-    // Force the writing of generated TS into output/YYYYMMDD-HHSSeco/ts-generator/thermal/mc-0
+    // Force the writing of generated TS into output/YYYYMMDD-HHSSeco/ts-generator/thermal
```

Comment:
Revert comment to original (still mc-0)

Reply:
It's not mc-0 if we generate thermal series through the tool

Reply:
So then `output/YYYYMMDD-HHSSeco/ts-generator/thermal[/mc-0]`